app/config/test_checker.py

```diff
@@ -0,0 +1,21 @@
+from django.test import Client
+from django.test import TestCase
+
+
+class CheckerUrlTestCase(TestCase):
```

Comment:
Does not need `django.test` here as no interaction with the DB is necessary.

So just this would be enough:
```py
from django.test import Client


def test_checker_url():
    #....
```
Any concerns with that?

Reply:
I thought that inheriting from TestCase is necessary :thinking:  Mabye there would be a better way without the Django baggage?

Reply:
Changed it to use `unittest.TestCase` directly :+1: